log4j-slf4j2-impl/src/main/java/org/apache/logging/slf4j/Log4jEventBuilder.java

```diff
@@ -112,7 +112,7 @@ public void log() {
                 .withMarker(marker)
                 .withThrowable(throwable);
         if (logBuilder instanceof BridgeAware) {
-            String fqcn = fqcnOverride != null ? fqcnOverride : FQCN;
+            String fqcn = this.fqcn != null ? this.fqcn : FQCN;
```

Comment:
`this.fqcn` can never be `null`, since calling `setCallerBoundary(null)` makes no sense.

Reply:
done